### ISSUE TITLE
duplicate results with mmf_predict

### ISSUE THREAD

**Issue by sallypannn** [facebookresearch/mmf#908]
Hi,

When I use mmf_predict on a trained model and a few test samples, I noticed that the number of samples must be a multiple of 4. If it is not, the model will output duplicate results for the first 1-3 samples. 

For example, if my test set contains 81 samples, then the model will output 84 rows and the first 3 samples will appear twice.

Could you please help me with this problem? Thanks!

**Comment by apsdehal** [comment]
Hi,
this is expected due to wrap around that happens in distributed settings on the dataloader. Please try rerunning on single GPU by setting environmental variable `CUDA_VISIBLE_DEVICES=0`.

**Comment by sallypannn** [comment]
Got it, thanks!
